templates/myaccount/form-edit-account.php

```diff
@@ -19,7 +19,7 @@
 
 do_action( 'woocommerce_before_edit_account_form' ); ?>
 
-<form class="woocommerce-EditAccountForm edit-account" action="" method="post">
+<form class="woocommerce-EditAccountForm edit-account" action="" method="post"<?php do_action( 'woocommerce_edit_account_form_tag' ); ?>>
```

Comment:
Just a small code style tweak please on both of these forms: I would like to see a space between the opening/closing PHP tags. `post" <?php ... ?> >`.